test/parallel/test-http2-compat-serverresponse-end.js

```diff
@@ -217,7 +219,7 @@ const {
   const server = createServer(mustCall((request, response) => {
     response.writeHead(HTTP_STATUS_OK, { foo: 'bar' });
     response.stream.on('close', mustCall(() => {
-      response.end(mustCall());
+      response.end(mustNotCall());
```

Comment:
This change is incorrect... the cb should still fire even after the underlying stream closes.